core/src/main/java/tc/oc/pgm/inventory/ViewInventoryMatchModule.java

```diff
@@ -177,7 +174,10 @@ public void updateMonitoredClick(final InventoryClickEvent event) {
           continue invLoop;
 
         for (int i = 0; i < inventory.getViewers().size(); i++) {
-          if (!inventory.getViewers().get(i).equals(tracker.getWatched().getViewers().get(i))) {
+          if (!inventory
```

Comment:
Not related to your changes but we should see about simplifying this in the future